kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
```

Comment:
`after uploaded file` -> `when a file is uploaded`.

Reply:
Done.